### ISSUE TITLE
Error in comment

### ISSUE THREAD

**Issue by jcubic** [eslint/eslint#8782]
**Tell us about your environment**

* **ESLint Version:** 3.14.1
* **Node Version:** v7.8.0
* **npm Version:** 4.2.0

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

```
"eslintConfig": {
    "parserOptions": {
      "ecmaVersion": 3,
      "sourceType": "script",
      "ecmaFeatures": { }
    },
    "env": {
      "browser": true,
      "jasmine": true,
      "node": true
    },
    "globals": {
      "jQuery": true,
      "sprintf": true,
      "$": true
    },
    "rules": {
      "eqeqeq": "error",
      "curly": "error",
      "no-unreachable": "error",
      "valid-typeof": "error",
      "no-unexpected-multiline": "error",
      "no-regex-spaces": "error",
      "no-irregular-whitespace": "error",
      "no-invalid-regexp": "error",
      "no-inner-declarations": "error",
      "no-func-assign": "error",
      "no-extra-semi": "error",
      "no-extra-boolean-cast": "error",
      "no-debugger": "error",
      "no-dupe-args": "error",
      "no-dupe-keys": "error",
      "no-duplicate-case": "error",
      "no-empty-character-class": "error",
      "no-ex-assign": "error",
      "no-func-assign": "error",
      "no-inner-declarations": "error",
      "array-callback-return": "error",
      "no-case-declarations": "error",
      "guard-for-in": "error",
      "no-caller": "error",
      "no-empty-function": "error",
      "no-eval": "error",
      "no-extend-native": "error",
      "no-extra-bind": "error",
      "no-fallthrough": "error",
      "no-global-assign": "error",
      "no-implicit-globals": "error",
      "no-implied-eval": "error",
      "no-multi-spaces": "error",
      "no-new-wrappers": "error",
      "no-redeclare": "error",
      "no-self-assign": "error",
      "no-return-assign": "error",
      "no-self-compare": "error",
      "no-throw-literal": "error",
      "no-unused-labels": "error",
      "no-useless-call": "error",
      "no-useless-escape": "error",
      "no-void": "error",
      "no-with": "error",
      "radix": "error",
      "wrap-iife": ["error", "inside"],
      "yoda": ["error", "never"],
      "no-catch-shadow": "error",
      "no-delete-var": "error",
      "no-label-var": "error",
      "no-undef-init": "error",
      "no-unused-vars": "error",
      "no-undef": "error",
      "comma-style": ["error", "last"],
      "comma-dangle": ["error", "never"],
      "comma-spacing": ["error", {"before": false, "after": true}],
      "computed-property-spacing": ["error", "never"],
      "consistent-this": ["error", "self"],
      "eol-last": ["error", "always"],
      "func-call-spacing": ["error", "never"],
      "key-spacing": ["error", {
        "beforeColon": false,
        "afterColon": true,
        "mode": "strict"
      }],
      "max-len": ["error", 90],
      "max-statements-per-line": "error",
      "new-parens": "error",
      "no-array-constructor": "error",
      "no-lonely-if": "error",
      "no-mixed-spaces-and-tabs": "error",
      "no-multiple-empty-lines": "error",
      "no-new-object": "error",
      "no-tabs": "error",
      "no-trailing-spaces": "error",
      "no-whitespace-before-property": "error",
      "object-curly-spacing": ["error", "never"],
      "space-before-blocks": "error",
      "keyword-spacing": ["error", {"before": true, "after": true}],
      "space-in-parens": ["error", "never"],
      "space-infix-ops": "error",
      "space-before-function-paren": ["error", "never"],
      "complexity": ["error", {"max": 25}],
      "indent": ["error", 4, {"SwitchCase": 1}],
      "linebreak-style": ["error", "unix"],
      "semi": ["error", "always"]
    }
  }
```

**What did you do? Please include the actual source code causing the issue.**

```js
/* global location, jQuery, setTimeout, window, global, localStorage, sprintf,
 *        setImmediate, IntersectionObserver, MutationObserver
 */
```

**What actually happened? Please include the actual, raw output from ESLint.** it show error in comment:
```
./node_modules/.bin/eslint test.js

/home/kuba/projects/jcubic/terminal/repo/test.js
  2:2  error  '*' is defined but never used  no-unused-vars

✖ 1 problem (1 error, 0 warnings)
```

**Comment by jcubic** [comment]
OK global is eslint directive, closing.
